chore: reducing ILP writer idle timeout defaults (#1063)
questdb/questdb.io: docs/reference/configuration.md

| line.tcp.connection.pool.capacity          | 64           | The maximum amount of pooled connections this interface may have.                                                                                                                                                                                     |
| line.tcp.timestamp                         | n            | Input timestamp resolution. Possible values are `n`, `u`, `ms`, `s` and `h`.                                                                                                                                                                          |
| line.tcp.msg.buffer.size                   | 32768        | Size of the buffer read from queue. Maximum size of write request, regardless of the number of measurements.                                                                                                                                          |

> | line.tcp.maintenance.job.interval          | 1000         | Maximum amount of time (in milliseconds) between maintenance jobs, these will commit any uncommitted data on inactive tables.                                                                                                                         |

Reviewer: ```suggestion
| line.tcp.maintenance.job.interval          | 1000         | Maximum amount of time (in milliseconds) between maintenance jobs committing any uncommitted data on inactive tables.                                                                                                                         |
```

> | line.tcp.min.idle.ms.before.writer.release | 500          | Minimum amount of idle time before a table writer is released in milliseconds.                                                                                                                                                                        |

Reviewer: ```suggestion
| line.tcp.min.idle.ms.before.writer.release | 500          | Minimum amount of idle time (in milliseconds) before a table writer is released.                                                                                                                                                                        |
```
